mkdocs.yml

```diff
@@ -64,7 +64,16 @@ markdown_extensions:
   - toc:
       permalink: true
 nav:
-  - Introduction: index.md
+  - Getting Started:
+    - Introduction: README.md
+    - Launching OVN-Kubernetes on Kubernetes: installation/launching-ovn-kubernetes-on-kubernetes.md
+    - Launching OVN-Kubernetes on KIND: installation/launching-ovn-kubernetes-on-kind.md
+    - Deploying Workloads on OVN-Kubernetes cluster: getting-started/example-pod-creation.md
+    - Deploying Services on OVN-Kubernetes cluster: getting-started/example-service-creation.md
+    - Setup and Building: getting-started/building-ovn-kubernetes.md
+    - Run OVN-Kubernetes From Release Image: getting-started/running-release.md
+    - CLI Guide: getting-started/cli-guide.md
+    - Configuration Guide: getting-started/configuration.md
   - Overview:
```

Comment:
Overview should be first, not sure why Getting started is moving positions here

Reply:
I had getting started in the first position because it contained the index/homepage for the project since I deleted the introduction subheading.

Reply:
I see, I don't think we need overview in getting started so we can keep things where they are in that case at least that's the change I had proposed in my PR for the release cut, but ofc I don't have strong opinions either ways.

Reply:
I felt that it was better for UI simplification since right now the index page gets its own subheading, which seems a little wasteful. Maybe instead we can just remove the subheading and keep everything else where it is?

Reply:
yeah that's the bit I did in my PR PTAL, if you rebase this we can have another round of reviews
the other thing I was thinking is.. is there any use to keep the deprecated docs at all?